### ISSUE TITLE
Model Load - KeyError: 'content-disposition'

### ISSUE THREAD

**Issue by maciekmasiul** [TheLastBen/fast-stable-diffusion#2782]
I have a problem when charging the model, a few days ago everything worked without a problem.

KeyError                                  Traceback (most recent call last)
Cell In[7], line 18
     12 MODEL_LINK = "https://drive.google.com/file/d/1Pi9E3m4FkUGGD-d_NuFyWl4xwflgeZ_t/view?usp=sharing"
     14 # A direct link to a Model or a shared gdrive link.
     15 
     16 
     17 #--------------------
---> 18 model=mdl(Original_Model_Version, Path_to_MODEL, MODEL_LINK)

File /workspace/mainrunpodA1111.py:172, in mdl(Original_Model_Version, Path_to_MODEL, MODEL_LINK)
    170       print('Model already exists')
    171 elif src=='gdrive':
--> 172    modelname=get_name(MODEL_LINK, True)
    173    model=f'/workspace/sd/stable-diffusion-webui/models/Stable-diffusion/{modelname}'
    174    if not os.path.exists(model):

File /workspace/mainrunpodA1111.py:488, in get_name(url, gdrive)
    486 res = requests.session().get(lnk, headers=headers, stream=True, verify=True)
    487 res = requests.session().get(get_url_from_gdrive_confirmation(res.text), headers=headers, stream=True, verify=True)
--> 488 content_disposition = six.moves.urllib_parse.unquote(res.headers["Content-Disposition"])
    489 filenm = re.search(r"filename\*=UTF-8''(.*)", content_disposition).groups()[0].replace(os.path.sep, "_")
    490 return filenm

File /usr/local/lib/python3.10/dist-packages/requests/structures.py:52, in CaseInsensitiveDict.__getitem__(self, key)
     51 def __getitem__(self, key):
---> 52     return self._store[key.lower()][1]

KeyError: 'content-disposition'

**Comment by ammartaher** [comment]
I have the same issue, did you figure it out by any chance?

**Comment by maciekmasiul** [comment]
> I have the same issue, did you figure it out by any chance?

Unfortunately, no.

**Comment by TheLastBen** [comment]
fixed, use the latest notebook, if it still doesn't work, try a different gdrive link

**Comment by ammartaher** [comment]
@TheLastBen It still didn't work for me. But just before I dive deep into it too, Would this be the solution to my suffer of having to download and wait for my models everytime from CivitAi? or is this also gonna basically take time downloading it from the gdrive? (I checked in the .py file this is what is happening if i am not mistaken)
Is there any way to keep the models downloaded just once and then use it right away ? 
Thanks alot :))

**Comment by maciekmasiul** [comment]
> fixed, use the latest notebook, if it still doesn't work, try a different gdrive link

Thanks for your reply.
It doesn't work for me either, I tried different links, uploaded all my models again so I guess that's not the problem.

**Comment by TheLastBen** [comment]
have you used this link for the notebook https://colab.research.google.com/github/TheLastBen/fast-stable-diffusion/blob/main/fast_stable_diffusion_AUTOMATIC1111.ipynb

**Comment by ammartaher** [comment]
> have you used this link for the notebook https://colab.research.google.com/github/TheLastBen/fast-stable-diffusion/blob/main/fast_stable_diffusion_AUTOMATIC1111.ipynb

I am using the runpod template to run ComfyUI, not the A1111. 
Sorry I wasn't clear in my comment on this post.

**Comment by maciekmasiul** [comment]
> have you used this link for the notebook https://colab.research.google.com/github/TheLastBen/fast-stable-diffusion/blob/main/fast_stable_diffusion_AUTOMATIC1111.ipynb

No, I use the Runpod service. Sorry, I actually did not mention that.

**Comment by TheLastBen** [comment]
will fix runpod shortly

**Comment by TheLastBen** [comment]
fixed

**Comment by TheLastBen** [comment]
> > have you used this link for the notebook https://colab.research.google.com/github/TheLastBen/fast-stable-diffusion/blob/main/fast_stable_diffusion_AUTOMATIC1111.ipynb
> 
> I am using the runpod template to run ComfyUI, not the A1111. Sorry I wasn't clear in my comment on this post.

if you don't remove the pod, the models will stay
